pcs/constraint.py

```diff
@@ -926,9 +926,11 @@ def _show_location_rules(
     return all_lines
 
 
-def _verify_node_name(node, existing_nodes):
+def _verify_node_name(node, existing_nodes, warn: bool = False):
     report_list = []
     if node not in existing_nodes:
+        if warn :
+            return False
```

Comment:
* This function returns a list of ReportItem objects. It must never return `False`.
* Instead of calling a function and passing it an argument to tell the function to do nothing, it is better not to call the function at all.
* The function doesn't seem to be called with `warn` parameter anywhere. What's the parameter purpose than?